realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncConfigurationTests.kt

```diff
@@ -366,10 +367,12 @@ class SyncConfigurationTests {
                 SyncConfiguration.defaultConfig(user, null as Int?),
                 SyncConfiguration.defaultConfig(user, null as Long?),
                 SyncConfiguration.defaultConfig(user, null as ObjectId?),
+                SyncConfiguration.defaultConfig(user, null as UUID?),
                 SyncConfiguration.Builder(user, null as String?).build(),
                 SyncConfiguration.Builder(user, null as Int?).build(),
                 SyncConfiguration.Builder(user, null as Long?).build(),
-                SyncConfiguration.Builder(user, null as ObjectId?).build()
+                SyncConfiguration.Builder(user, null as ObjectId?).build(),
+                SyncConfiguration.Builder(user, null as UUID?).build()
```

Comment:
Shouldn't we also have a test of actually syncing with a `UUID` partition value? and not only nulls

Reply:
Right now we only have full integration tests with String partition keys...we don't test Integer or ObjectId either as it requires a complete test app just for that.

Reply:
Ok. Have you verified it manually? I previously experienced issues with the core implementation not having correct round trip on some of the BsonBinary subtypes (https://github.com/realm/realm-core/issues/4186). It was a long time ago, so probably fixed in one of their updates, but just in case.